.ci/download_release_and_verify_checksum.sh

```diff
@@ -14,7 +14,11 @@ echo "Downloading artifacts for tag \'${release_tag}\' to \'${downloaded_package
 mkdir -p "${downloaded_packages_location}"
 pushd "${downloaded_packages_location}"
 
-"${this_script_dir}/run_command_with_timeout_and_retries.sh" --timeout=10 --max-tries=3 --wait-time-before-retry=10 --retry-on-error=yes -- gh release download "${release_tag}"
+run_command_with_timeout_and_retries_args=(--retry-on-error=yes)
+run_command_with_timeout_and_retries_args=(--max-tries=3 "${run_command_with_timeout_and_retries_args[@]}")
```

Comment:
IIRC, arrays in bash might be only available for a particular bash version onwards, will it be possible to use?

```suggestion
run_command_with_timeout_and_retries_args=(--max-tries=3 "${run_command_with_timeout_and_retries_args}")
```


IIUC,`run_command_with_timeout_and_retries_args` appends its previous value and a new value, so the `[@]` won't be needed, correct?

Reply:
Maybe I misunderstood what you mean but your suggestion doesn't work in my environment:
```
$ echo "${BASH_VERSION}"
5.0.17(1)-release

$ a=(1); a=("${a[@]}" 2); a=("${a[@]}" 3); echo "${a[@]}"
1 2 3

$ a=(1); a=("$a" 2); a=("$a" 3); echo "${a[@]}"
1 3
```

Reply:
Aha! It is not a variable but an array. IIRC arrays are not available for bash version < 4.0

As long as all those docker images use bash version >= 4 then it is safe to use